### DISCUSSION TITLE
Can't create Productions

### DISCUSSION THREAD

**Discussion by squinonescolon** [cgwire/kitsu#774] [Q&A]
Hi guys,
I installed the self-hosted version via your instructions(great write up by the way) and I ended up with a site and no errors as far as I could see. I'm having a problem where I can't create a production. Seems like all the other features work, I can add tasks, shots, assets, etc, but the new production page seems to be broken. I add all the requisites and I get a "All done, Let's go", but when I click I get an error: "An error occurred while creating the production " This is what I can get in errors on the page:

Firefox can’t establish a connection to the server at ws://localhost/socket.io/?EIO=3&transport=websocket&sid=ae5a9e65dadc4f4195febd3c3f011604. websocket.js:124:12
The connection to ws://localhost/socket.io/?EIO=3&transport=websocket&sid=ae5a9e65dadc4f4195febd3c3f011604 was interrupted while the page was loading. websocket.js:124:12
TypeError: e.productionStatus[0] is undefined
    t NewProduction.vue:850
    l runtime.js:63
    k runtime.js:294
    A runtime.js:119
    Babel 4
    createProduction NewProduction.vue:842
    VueJS 29
    input NewProduction.vue:12
    VueJS 4
    updateValue Combobox.vue:96
    VueJS 3
 undefined NewProduction.vue:867

**Comment by frankrousseau** [comment]
Did you init data on your instance properly? It seems that production status were not created during the installation.

**Comment by squinonescolon** [comment]
It's probable that I didn't. What step should I back track to?

**Comment by frankrousseau** [comment]
Did you run the command below?

```DB_PASSWORD=yourdbpassword zou init-data```

**Comment by squinonescolon** [comment]
Yes I did. I don’t have any output now, since it was a couple of days ago. But everything seem to have gone fine. Any way to troubleshoot that?

—
Https://www.imdb.com/name/nm1167131
Https://vimeo.com/stevenquinones
http://stevenquinones.com/resume.html

**Comment by frankrousseau** [comment]
Can you tell me what returns the following URL: http://localhost:8080/api/data/project-status ?

**Comment by squinonescolon** [comment]
I get Page not found.

**Comment by squinonescolon** [comment]
Sorry, mistyped it, the answer is unable to connect

**Comment by frankrousseau** [comment]
Sorry I mean:  https://your.kitsu.url/api/data/project-status 

Where you must change your.kitsu.url by the URL you use to access your Kitsu.

**Comment by squinonescolon** [comment]
I’m using it in localhost:

http://localhost/login/api/data/project-status

I get:
JSON heading is empty,
the Raw Data heading is [],
the Headers is pasted below:

—
Https://www.imdb.com/name/nm1167131
Https://vimeo.com/stevenquinones
http://stevenquinones.com/resume.html

**Comment by frankrousseau** [comment]
So it seems that your data are still not properly initialised. It think you should reset the database fully:

```
DB_PASSWORD=yourdbpassword zou clear-db
DB_PASSWORD=yourdbpassword zou upgrade-db
DB_PASSWORD=yourdbpassword zou init-data
```

Make sure too that your environment variables are properly loaded especially the one that gives your database name.

**Comment by squinonescolon** [comment]
So I decided to install from scratch and it’s working now! Thanks so much for you troubleshooting advice. Another problem I do have is I have Houdini HQueue(a web server based render farm application) installed in that same machine and I can’t have both HQueue and Kitsu running at the same time. Can you suggest settings so I can have Kitsu and HQueue coexist? I would prefer to modify the Zou/Kitsu configuration if possible. HQueue is a bear to get to work and I don’t want to touch it. Thanks in advance.
Best regards,
Steven

—
Https://www.imdb.com/name/nm1167131
Https://vimeo.com/stevenquinones
http://stevenquinones.com/resume.html

**Reply by frankrousseau** [reply]
Sorry, I forgot to answer. I don't know how HQueue is installed. But the problem is probably that they try both to use the 80 port. With Nginx and two different domain names you can make them live together.
